docs/core/deploying/native-aot/diagnostics.md

```diff
@@ -62,7 +62,7 @@ You can launch a Native AOT-compiled executable under the Visual Studio debugger
 
 To set a breakpoint that breaks whenever an exception is thrown, choose the **Breakpoints** option from the **Debug > Windows** menu. In the new window, select **New > Function** breakpoint. Specify `RhThrowEx` as the Function Name and leave the Language option at **All Languages** (don't select C#).
```

Comment:
There is one more mention of register names above. Could you please clarify it as well?

The exception is stored in the `rcx` or `x0` register. -> The exception is stored in the first argument register that is `rcx` on x64 and `x0` on Arm64.

Reply:
Done